<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>298</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Projectboards:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="pb_list">
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string>&amp;New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_imp">
         <property name="text">
          <string>&amp;Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ren">
         <property name="text">
          <string>Rena&amp;me</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rem">
         <property name="text">
          <string>&amp;Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
